### ISSUE TITLE
`get_tf_tensor_data` in utils.py cannot get tensor content when the content is not stored in `tensor_content`

### ISSUE THREAD

**Issue by ziyuang** [onnx/tensorflow-onnx#465]
I have a tensor like this

```
dtype: DT_HALF
tensor_shape {
  dim {
    size: 64
  }
}
half_val: 9973
half_val: 43183
half_val: 14398
half_val: 47709
...
half_val: 13378
```
Here the content is not stored in `tensor_content` so both `get_tf_tensor_data`  and `tf_to_onnx_tensor` will think it is an empty tensor then reshaping an empty numpy array to size 64 will raise an error.

**Comment by ziyuang** [comment]
As for "other errors", I've got this after adding those two lines:

`ERROR:tf2onnx:node scope/Slice: exception get tensor value: scope/Shape_4__103 must be Const`
I did some printing and the node is of type `Cast` with a value `None`.

**Comment by guschmue** [comment]
Yes, looks that your 2 lines are missing - thanks for catching. If you like please send a PR, else we can add it.
The [slice before opset10](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Slice-1) - yes, the slice in onnx has start, end, axis as attributes so tf2onnx need to find this as const.
This is going to change with [slice in opset10](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Slice)

**Comment by ziyuang** [comment]
> Yes, looks that your 2 lines are missing - thanks for catching. If you like please send a PR, else we can add it.
> The [slice before opset10](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Slice-1) - yes, the slice in onnx has start, end, axis as attributes so tf2onnx need to find this as const.
> This is going to change with [slice in opset10](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Slice)

Is opset10 ready in the master?

**Comment by guschmue** [comment]
opset10 is no released (coming in the next few days). Once released it takes maybe 2 weeks before it complete enough in tf2onnx.

**Comment by guschmue** [comment]
For half_val: merged your PR https://github.com/onnx/tensorflow-onnx/pull/467

**Comment by nbcsm** [comment]
i guess this issue is resolved.
feel free to re-open if there is further issue.
